app/dashboard/projects/[projectId]/ActionPanel.tsx

```diff
@@ -0,0 +1,73 @@
+'use client'
```

Comment:
If this is just work for webhook settings. Rename this file to `WebhookSettingsPannel` or something like this.